### DISCUSSION TITLE
제페토 상점 아이템 중에 인형의 경우 최대 폴리곤 수는 어떻게 되나요?

### DISCUSSION THREAD

**Discussion by ellejou76** [naverz/zepeto-studio-kor#813] [5. Q&A]
안녕하세요.

의상 아이템을 제작 중인데, 상점에 보면, 등에 매달린 큰 인형아이템을 제작 하려고 하는데.
관련해서 아래 가이드를 체크해봐도 최대 폴리곤 수를 알 수가 없어서 문의드립니다.

이 경우는 아래 링크속의 정보 중에 어떤 아이템을 참고해야 할까요?
https://studio.zepeto.me/kr/guides/item-guidelines

감사합니다.

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, 이곳은 제페토 월드 개발 커뮤니티입니다.
아이템 제작과 관련된 문의 사항은 제페토 고객센터를 이용해 주시기를 부탁 드립니다.
https://support.zepeto.me/hc/ko

감사합니다.
